src/lt/objs/sidebar/workspace.cljs

```diff
@@ -284,11 +285,32 @@
                         (files/mkdir final-path)
                         (object/raise folder :start-rename!))))
 
+(defui rename-input [this]
+  [:input.rename {:type "text" :value (:path @this) :style {:width "100%"}}]
+  :focus (fn []
+           (object/raise this :rename.focus))
+  :blur (fn []
+          (object/raise this :rename.blur)))
+
+(object/object* ::rename-dialog
+                :tags #{:tree.rename}
+                :max-height 130
+                :path ""
+                :renaming? true
+                :behaviors #{::rename-focus ::rename-blur ::rename-submit ::rename-cancel ::rename-close! ::rename-file ::rename-folder}
```

Comment:
@kenny-evitt, silly question, but is there a way to not specify the behaviours for the object and simply raise whatever behaviour? 

My question stems from the ::workspace.folder and ::workspace.file objects as they don't seen to have behaviours specified and work with various other behaviours... I could just be misunderstanding (probably am), so feel free to tell me otherwise. 

This is related to the PR as the previous rename functionality had two behaviours, rename-file rename-folder, both with the same trigger. Specifying the behaviours in the rename-dialog object is causing both rename-file and rename-folder to trigger as a result, which is not currently desired.